opencti-platform/opencti-front/src/private/components/analyses/reports/ReportKnowledgeGraph.jsx

```diff
@@ -904,6 +904,36 @@ class ReportKnowledgeGraphComponent extends Component {
     this.setState({ numberOfSelectedNodes: this.selectedNodes.size });
   }
 
+  handleSelectRelationshipsByAdjacentNode(type) {
+    const selectedNodes = Array.from(this.selectedNodes);
+    const selectedNodesIds = R.map((n) => n.id, selectedNodes);
```

Comment:
```suggestion
    const selectedNodesIds = selectedNodes.map((n) => n.id);
```
(we avoid using Ramda when possible)